### ISSUE TITLE
Stuck on "Loading Southwest Check-In page"

### ISSUE THREAD

**Issue by HsuTimothy** [jdholtz/auto-southwest-check-in#170]
### Version

v6.1

### Description

Hi! 

I'm unsure if this is a user error but I'm getting stuck on the "Loading Southwest Check-In page" step. For a bit of context, I'm running the vanilla script from my local machine. 

My chrome version is 117.

Let me know if there's any other info I can help provide!

### To Reproduce

1. Run `python southwest.py CONFIRMATION FIRST LAST -v`
2. Script hangs at "Loading Southwest Check-In page"

### Expected Behavior

Should schedule check-in

### Relevant logs and program output

```sh
2023-10-05 13:35:48 DEBUG MainProcess[log:23]: Initialized the application
2023-10-05 13:35:48 DEBUG MainProcess[main:112]: Auto-Southwest Check-In v6.1
2023-10-05 13:35:48 DEBUG MainProcess[main:113]: Called with 3 arguments
2023-10-05 13:35:48 DEBUG MainProcess[config:107]: Initializing configuration file
2023-10-05 13:35:48 DEBUG MainProcess[config:135]: Reading the configuration file
2023-10-05 13:35:48 DEBUG MainProcess[config:139]: No configuration file found. Using defaults
2023-10-05 13:35:48 DEBUG MainProcess[config:125]: Creating configurations for 1 reservations
2023-10-05 13:35:48 DEBUG MainProcess[main:137]: Reservation added through CLI arguments
2023-10-05 13:35:48 DEBUG MainProcess[main:142]: Monitoring 0 accounts and 1 reservations
2023-10-05 13:35:49 DEBUG Process-1[reservation_monitor:60]: Acquiring lock...
2023-10-05 13:35:49 DEBUG Process-1[reservation_monitor:62]: Lock acquired
2023-10-05 13:35:49 DEBUG Process-1[checkin_scheduler:50]: Refreshing headers for current session
2023-10-05 13:35:49 DEBUG Process-1[webdriver:124]: Starting webdriver for current session
2023-10-05 13:35:51 DEBUG Process-1[webdriver:135]: Loading Southwest Check-In page
```

**Comment by jdholtz** [comment]
Hey. Does it ever time out? Also, do you get the same result if you try with the Docker image?

**Comment by HsuTimothy** [comment]
> Hey. Does it ever time out? Also, do you get the same result if you try with the Docker image?

It does eventually time out. I haven't actually tried with docker (having trouble installing Docker and getting it to work on my M1 Mac 🤦‍♂️)

**Comment by jdholtz** [comment]
> It does eventually time out

Can you share what the traceback is when it times out?

**Comment by HsuTimothy** [comment]
> Can you share what the traceback is when it times out?

I'm actually not seeing a timeout right now (been running for the last hour) so perhaps the "timeout" I observed earlier might've just been me accidentally killing the process

**Comment by jdholtz** [comment]
Hm. I have a few suggestions you can try. First, on line 136 of `lib/webdriver.py`, can you add `driver.default_get(CHECKIN_URL)` instead of `driver.get(CHECKIN_URL)` and run the script again to see if it doesn't hang?

If it is still hanging, try to add `driver.set_page_load_timeout(15)` to the line before the `driver.get` (make sure to revert it back). Then see if it times out. Also, you can see what is going on if you replace `headless=True` with `headed=True` on line 128 of `lib/webdriver.py`.

**Comment by HsuTimothy** [comment]
I tried with `driver.default_get(CHECKIN_URL)` and I observed that it was still hanging.

I changed it back to `driver.get(CHECKIN_URL)` and then added the `driver.set_page_load_timeout(15)` and it timed out. I tried with `headed=True` and observed that actually the southwest page is just hanging 

![image](https://github.com/jdholtz/auto-southwest-check-in/assets/14950271/d0a9522b-de33-4aae-9b06-91f5417199b9)

However, if I put mobile.southwest.com/check-in into my google chrome (not created from script) the site loads just fine. 

Seems like the issue may actually not be with the script itself but moreso with the Southwest page?
